src/compiler/utilities.ts

```diff
@@ -26,20 +26,6 @@ namespace ts {
         return undefined;
     }
 
-    export function findDeclaration<T extends Declaration>(symbol: Symbol, predicate: (node: Declaration) => node is T): T | undefined;

```

Comment:
Actually, the rewrite at the bottom of the change to `find(symbol.declarations, ...` is good enough since my new usage requires a cast as well.

(I also just merged the new usage, so you will see it when you merge from master.)